### DISCUSSION TITLE
gallery tab [Apr 29]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#1988] [Discord Threads]
> 🙋‍♂️ **Guineu**

**Comment by o1lab** [comment]
> 👨‍🚀 **wingkwong**  
#### do u mean the left n right button?  
![image0](https://cdn.discordapp.com/attachments/968875461121835048/969094814970826812/unknown.png)  
#  
> 🙋‍♂️ **Guineu**  
#### Nope, I mean from one field to another. Enter doesn't go to the next one either.  
![image0](https://cdn.discordapp.com/attachments/968875461121835048/969119757406183444/unknown.png)  
#  
> 👨‍🚀 **wingkwong**  
#### this is Form View  
#  
> 👨‍🚀 **wingkwong**  
#### please create an issue in github  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/968875461121835048)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
